docs/decisions/app/0004-uswds-in-react.md

```diff
@@ -0,0 +1,70 @@
+# U.S. Web Design System in React
+
+- Status: proposed
+- Deciders: [list everyone involved in the decision] <!-- optional -->
+- Date: 2022-12-05
+
+Technical Story: #19
+
+## Context and Problem Statement
+
+- The U.S. Web Design System (USWDS) only provides HTML and CSS for its components. It includes a small bit of vanilla JS to add interactivity to some components like the date picker.
+- It's common for projects to write their own React components to output the USWDS HTML, to reduce the amount of boilerplate needed to use the USWDS components.
+- [Previous research by Kalvin and Shannon](https://docs.google.com/document/d/1KRWzH_wJUPKkFmBlxj6SM2yN3W7Or89Wa4TBVM3Ksog/edit) discovered that Nava engineers and designers universally agreed that being able to use a React USWDS component library when starting new projects would be valuable.
+
+## Considered Options
+
+- Use the existing open source [`react-uswds` library](https://github.com/trussworks/react-uswds)
+- Create our own React USWDS component library
+- Leave the responsibility to each project team
+
+## Decision Outcome
+
+Use the existing open source [`react-uswds` library](https://github.com/trussworks/react-uswds), to avoid reinventing the wheel and because it's overall a well-built and maintained library.
+
+## Pros and Cons of the Options <!-- optional -->
```

Comment:
Do we just leave the optional bit in? (I guess kind of a newb markdown question on my part - I usually delete it)

Reply:
I can clean it up - copied over from the template